.github/workflows/scheduled_test_cpu.yml

```diff
@@ -19,9 +19,14 @@ jobs:
       fail-fast: false
       matrix:
         os: ['Ubuntu', 'Windows', 'MacOS']
-        python-version: ['3.7', '3.8']
-        pytorch-version: ['1.9.1', '1.10.2', '1.12.1', '1.13.1']
+        python-version: ['3.7', '3.8', '3.9', '3.10']
+        pytorch-version: ['1.9.1', '1.10.2', '1.11.0', '1.12.1', '1.13.1']
```

Comment:
Just as note: I don't know why torch 1.11 is out of this list, I guess I just forgot when I added this workflow 😅